client/src/commons/Contexts/ExposuresAndFlights.ts

```diff
@@ -31,6 +35,10 @@ export const fieldsNames = {
 };
 
 export const initialExposuresAndFlightsData: ExposureAndFlightsDetails = {
+    exposures: []
```

Comment:
add type (like in exposures array in ExposureAndFlightsDetails)

Reply:
I cant add a type because this is an object and not a type